modules/007-registrypackages/images/containerd-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip container-selinux.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="container-selinux.x86_64.rpm"
+packages_to_install=""
+package_container_selinux="container-selinux"
+package_file_container_selinux="${package}.x86_64.rpm"
+rpm_name_container_selinux="$(rpm -qip ${package_file_container_selinux} 2>/dev/null | grep Name | awk '{print $3}')"
+rpm_version_container_selinux="$(rpm -qip ${package_file_container_selinux} 2>/dev/null | grep Version | awk '{print $3}')"
+
+if ! rpm --quiet -q "${rpm_name_container_selinux}-${rpm_version_container_selinux}"; then
+  packages_to_install="${packages_to_install} ${package_file_container_selinux}"
```

Comment:
`${packages_to_install}` is always empty here. You should remove it or it indicates that you have error in logic or maybe i skipped logic

Reply:
oh, i see same logic below i think you can stay it